packages/portal/swagger-ui-web-component/package.json

```diff
@@ -14,8 +14,8 @@
   },
   "dependencies": {
     "@kong/swagger-ui-kong-theme-universal": "^4.3.3",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
```

Comment:
@davidma415 I had to bump these packages to fix build error: https://github.com/Kong/public-ui-components/actions/runs/9099234899/job/25011319267
Not sure if this is ideal

Reply:
No I don't think this is ideal; if we bump the major, I believe it would be a major version bump for our package as well.

It was building fine before this PR, what changed that now causes it to fail and require a version bump?